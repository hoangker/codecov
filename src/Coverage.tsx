import React, { Component } from "react";

function Coverage() {
  return (
    <div>
      <span>Hello World!</span>    
     </div>
    );
}

export default Coverage;
