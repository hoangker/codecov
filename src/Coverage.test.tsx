/**
 * Test we can create the Contact component
 */
import React from "react";
import { create } from "react-test-renderer";
import Coverage from "./Coverage";
describe("Coverage component", () => {
  test("it matches the snapshot", () => {
    const component = create(<Coverage/>); 
    expect(component.toJSON()).toMatchSnapshot();
  });
});
