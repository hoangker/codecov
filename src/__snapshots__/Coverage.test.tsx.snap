// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Coverage component it matches the snapshot 1`] = `
<div>
  <span>
    Hello World!
  </span>
</div>
`;
